include/database.py

```diff
@@ -1,18 +1,70 @@
 # database.py
 #
 # This module provides a wrapper for basic database functionality.
+import sys
 import psycopg2
 
+
 def select(connectionString, sql, parameters):
+
     # Open the connection
-    connection = psycopg2.connect(connectionString)
-    cursor = connection.cursor()
+    try:
+        # connect_timeout for connectionString is 60 in the yml file
+        connection = psycopg2.connect(connectionString, connect_timeout = 5)
+        cursor = connection.cursor()
+
+        # Execute the query, note the rows
+        cursor.execute(sql, parameters)
+
+        rows = cursor.fetchall()
+
+        # Clean-up and return
+        cursor.close()
+        connection.close()
+
+        return rows
+
+
+    # operational error is raised when parameters passed to connect() are incorrect
+    # or if the server runs out of memory
```

Comment:
Sure, but you might want to note that connection timeouts also generate the same error.